### PR TITLE
Terminal: Fix crash for 'printf "\x1b[1;1r"'

### PR DESCRIPTION
**Pull request by trufae**

### CODE REVIEW

File: Userland/Libraries/LibVT/Terminal.cpp [hunk 1]
```diff
@@ -743,6 +746,9 @@ void Terminal::scroll_down(size_t count)
 // Insert `count` blank lines at the bottom of the region. Text moves up, top lines get added to the scrollback.
 void Terminal::scroll_up(u16 region_top, u16 region_bottom, size_t count)
 {
+if (region_top > region_bottom) {
+return;
+}
```
Comment (alimpfard) [inline Userland/Libraries/LibVT/Terminal.cpp:751]: This bit isn't necessary, let's keep the assert for more bugs

**Review by alimpfard — CHANGES_REQUESTED** [review]
This is in LibVT, and the commit message is talking about fixing one of the symptoms, let's make that about the fix instead
e.g. "LibVT: Ignore DECSTBM with invalid values"

File: Userland/Libraries/LibVT/Terminal.cpp (hunk 1 above)
Comment (trufae) [reply Userland/Libraries/LibVT/Terminal.cpp:751]: it was for debugging, it wasnt suposed to be commited. you review too fast :D

**Comment by alimpfard** [comment]
uhhh...could you fixup the commit message so we could merge this?

**Comment by trufae** [comment]
The commit message can be changed with the squash button in github. I’m fine if you do so for me now. Im busy irl and will take some hours to get back to the shell

**Comment by alimpfard** [comment]
🤔 TIL...

**Pull request merged**